src/ogma/features/nodeGrouping.ts

```diff
@@ -79,6 +79,10 @@ export class NodeGroupingTransformation {
         duration: 300,
         padding: 10
       });
+      // TODO remove setTimeout when LKE-10453 is fixed
```

Comment:
Hopefully we will not forget about it, at least if possible to link Jira tickets together

Reply:
the ticket is also linked to the node grouping epic. We will close the epic only when it is fixed